src/Model/Photo.php

```diff
@@ -331,71 +331,71 @@ public static function createPhotoForExternalResource($url, $uid = 0, $mimetype
 	 * @return boolean True on success
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function store(Image $Image, $uid, $cid, $rid, $filename, $album, $scale, $type = self::DEFAULT, $allow_cid = "", $allow_gid = "", $deny_cid = "", $deny_gid = "", $desc = "")
+	public static function store(Image $image, int $uid, int $cid, string $rid, string $filename, string $album, int $scale, int $type = self::DEFAULT, string $allow_cid = '', string $allow_gid = '', string $deny_cid = '', string $deny_gid = '', string $desc = ''): bool
```

Comment:
I prefer to capitalize variables holding structured objects as they are in minority and I feel like they should stand out.
```suggestion
	public static function store(Image $Image, int $uid, int $cid, string $rid, string $filename, string $album, int $scale, int $type = self::DEFAULT, string $allow_cid = '', string $allow_gid = '', string $deny_cid = '', string $deny_gid = '', string $desc = ''): bool
```

Reply:
No, let's make all variables the same, please no extra rules. :-)

Reply:
How do you feel about it, @annando  and @nupplaphil ?